focus-android/automation/taskcluster/androidTest/ui-test.sh

```diff
@@ -98,10 +98,18 @@ function failure_check() {
     echo
     if [[ $exitcode -ne 0 ]]; then
         echo "FAILURE: UI test run failed, please check above URL"
+        TEST_STATUS="FAIL"
     else
 	      echo "All UI test(s) have passed!"
+        TEST_STATUS="PASS"
     fi
 
+    {
+        echo "TEST_STATUS=${TEST_STATUS}"
+        echo "PRODUCT_TYPE=${PRODUCT_TYPE}"
+        echo "RELEASE_TYPE=${RELEASE_TYPE}"
+    } >> test_dashboard.env
```

Comment:
I'd kind of prefer a variable more closely tuned to the data such as `execution_metadata.env` to be reflective of these three variables and what they are currently used for .

Reply:
done

Reply:
Looks like it needs updating in the focus `ui-test.sh` file